config/splat.us.stnz0.yaml

```diff
@@ -151,7 +151,8 @@ segments:
       - [0x30900, .rodata, e_stage_name]
       - [0x30934, .rodata, prim_helpers]
       - [0x3093C, .rodata, e_life_up]
-      - [0x30958, c]
+      - [0x30958, c, st_debug]
+      - [0x30A20, c, e_lock_camera]
```

Comment:
Split these while I was here, and refactored `EntityBackgroundBlock` to be named the same as its common implementation elsewhere